ZitiNativeApiForDotnetCore/msvc-build.bat

```diff
@@ -32,15 +32,23 @@ REM copy library/ZitiStatus.cs ../Ziti.NET.Standard/src/OpenZiti
 
 ECHO Build from cmake using: 
 ECHO     cmake --build %BUILDFOLDER%\x86 --config Debug
+cmake --build %BUILDFOLDER%\x86 --config Debug
```

Comment:
these were intentionally left out so that people would/could do this on their own. i don't want to invoke this for them. let's revert these two changes

Reply:
done